test/entity-store.js

```diff
@@ -171,6 +171,15 @@ suite('entity-store', function() {
 			}
 		});
 
+		test('remove rejects undefined entityid', async() => {
```

Comment:
Maybe we could have a test that does an actual remove too?

Reply:
done!